validator/schemas/entity-schema-v1.json

```diff
@@ -62,13 +62,30 @@
       ]
     }
   ],
-  "required": [
-    "domain",
-    "type",
-    "name",
-    "identifier"
+  "oneOf": [
+    {
+      "required": [
+        "domain",
+        "type",
+        "name",
+        "identifier"
+      ]
+    },
+    {
+      "required": [
+        "synthesize"
+      ]
+    }
   ],
   "properties": {
+    "synthesize": {
+      "$id": "#/properties/synthesize",
+      "type": "boolean",
+      "title": "Whether the entity should be synthesized",
```

Comment:
Maybe "the entity type" is more accurate. 

Also, if I read this correctly a non-synth'd entity would  not have the domain/type fields, which however seem relevant anyway?

```
synthesize: false
```


I'm wondering if we should do something like this instead.  Move the synthesis config inside a `synthesis` node:

```
domain: INFRA
type: ETCD_CLUSTER

synthesis:
  identifier: clusterName  
  name: clusterName
  conditions:
    - attribute: metricName
      prefix: etcd_
  tags:
    - label.topology.kubernetes.io/region
    - ...
```

This would let us define non-synthesisable fields as:

```
domain: INFRA
type: ETCD_CLUSTER
```

(e.g. absence of a synthesis node)

What do you think?

Reply:
The domain and type are also on the folder name, so I was counting on that (I thought we'd ignore the definition file), but what you propose seems like a really clean solution so I'm happy to go with it.